batch/batch/aioclient.py

```diff
@@ -141,11 +141,11 @@ def __init__(self, batch, job_id, attributes=None, parent_ids=None, _status=None
     async def is_complete(self):
         if self._status:
             state = self._status['state']
-            if state in ('Complete', 'Cancelled'):
+            if state in ('Error', 'Failed', 'Success', 'Cancelled'):
```

Comment:
Would it make it easier to store this set (and others) in some higher scope (potentially global), so that you don't need to re-type, and maybe miss one?

```python
if state in finished_set:
```
In no way blocking, just may make things less error prone as the number of state slices expands.

Reply:
great suggestion!